metricbeat/module/kubelet/container/data.go

```diff
@@ -0,0 +1,72 @@
+package container
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/kubelet"
+)
+
+func eventMapping(content []byte) ([]common.MapStr, error) {
+	events := []common.MapStr{}
+	var summary kubelet.Summary
+
+	err := json.Unmarshal(content, &summary)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot unmarshal json response: %s", err)
+	}
+
+	node := summary.Node
+	for _, pod := range summary.Pods {
+		for _, container := range pod.Containers {
+			containerEvent := common.MapStr{
+				mb.ModuleData: common.MapStr{
+					"pod": common.MapStr{
```

Comment:
schema wouldnt work because the input interface coming from the HTTP has arrays.